src/Microsoft.Azure.ServiceBus/Primitives/ConcurrentExpiringSet.cs

```diff
@@ -65,6 +65,13 @@ void CollectExpiredEntries()
                 }
             }
 
+            if (this.dictionary.Count <= 0)
+            {
+                return;
+            }
+
+            await Task.Delay(1000);
```

Comment:
>1000 [](start = 29, length = 4)

Represent this through `TimeSpan`. Its more readable.

Reply:
And 1000 is a small number. Maybe make it once in 30 seconds or so

---
In reply to: [191035182](https://github.com/Azure/azure-service-bus-dotnet/pull/473#discussion_r191035182) [](ancestors = 191035182)